src/main/java/org/folio/entlinks/domain/repository/AuthorityBaseCqlRepository.java

```diff
@@ -0,0 +1,14 @@
+package org.folio.entlinks.domain.repository;
+
+import java.util.UUID;
+import org.folio.entlinks.domain.entity.AuthorityBase;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+
+public interface AuthorityBaseCqlRepository<T extends AuthorityBase> {
+
+  Page<T> findByCql(String cql, Pageable pageable);
```

Comment:
There is ```findByCql(cql, pageable)``` also in ```JpaCqlRepository```, the idea was to hide/replace it or something else? maybe there is no need to extend JpaCqlRepository interface now?